lib/Crypt/LE.pm

```diff
@@ -671,7 +671,7 @@ sub _set_key {
     my $pem = $key->get_private_key_string;
     my ($n, $e) = $key->get_key_parameters;
     return $self->_status(INVALID_DATA, "Key modulus is divisible by a small prime and will be rejected.") if $self->_is_divisible($n);
-    $key->use_pkcs1_padding;
+    $key->use_pkcs1_oaep_padding;
```

Comment:
so technically oaep is default, removing the line would also work. The question is why did we set this to begin with. Any chance we could instead support ECDSA like Net::ACME2 does? 

https://metacpan.org/release/FELIPE/Net-ACME2-0.35/source/lib/Net/ACME2/AccountKey.pm#L73